.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "7.5.0"
+env:
+  - NPM_CONFIG_LOGLEVEL=warn
+install:
+  - npm install
+  - npm run bootstrap
```

Comment:
Good shout I just copied it over from `.drone.yml`. Let's see what happens in the [build](https://travis-ci.org/Skyscanner/backpack/builds/256812600)